sdk/nodejs/package.json

```diff
@@ -25,5 +25,8 @@
         "mocha": "^3.5.0",
         "node-gyp": "^3.6.2",
         "tslint": "^5.7.0"
+    },
+    "pulumi": {
+        "comment": "Do not remove. Marks this as as a deployment-time-only package"
```

Comment:
Should we depend on some specific sub-property being present in the `pulumi` bag?  It's not immediately clear that having the `pulumi` bag at all should indicate that this is deployment-time-only.  But moreso, it would potentially also avoid the need for this sort of `"comment"` to explain why an empty `"pulumi"` section is needed/meaningful.

Reply:
Well, i'd like to add a "runtimeDependencies: { }" section.  But then we'd still need a comment saying to not remove this as it's actually relevant.

Reply:
Ok.  As per our discussion, the presence of a "pulumi" section will, by default, mean: "do not include this package in closure serialization".  If we need an opt-out escape hatch in the future we can add a boolean to this to indicate that's what we want.